exporting/tests/test_exporting_engine.h

```diff
@@ -145,6 +149,16 @@ void __wrap_kinesis_put_record(
     size_t data_len);
 int __wrap_kinesis_get_result(void *request_outcomes_p, char *error_message, size_t *sent_bytes, size_t *lost_bytes);
 
+int __wrap_pubsub_init(
```

Comment:
Are we also missing `#if ENABLE_EXPORTING_PUBSUB` here?

Reply:
They are harmless declarations. We just don't use these functions in case pubsub is not enabled. There are no warnings when tests are built.